stripecardscan/src/main/java/com/stripe/android/stripecardscan/framework/api/dto/ClientStats.kt

```diff
@@ -10,43 +9,42 @@ import kotlinx.serialization.Serializable
 
 @Serializable
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-internal data class StatsPayload(
+data class StatsPayload(
     @SerialName("instance_id") val instanceId: String,
     @SerialName("scan_id") val scanId: String?,
     @SerialName("payload_version") val payloadVersion: Int = 2,
     @SerialName("device") val device: ClientDevice,
     @SerialName("app") val app: AppInfo,
     @SerialName("scan_stats") val scanStats: ScanStatistics,
-    @SerialName("model_versions") val modelVersions: List<ModelVersion>,
+// TODO: these should probably be reported as part of scanstats
+//    @SerialName("model_versions") val modelVersions: List<ModelVersion>,
 )
 
 @Serializable
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-internal data class ScanStatistics(
+data class ScanStatistics(
     @SerialName("tasks") val tasks: Map<String, List<TaskStatistics>>,
-    @SerialName("repeating_tasks") val repeatingTasks: Map<String, List<RepeatingTaskStatistics>>,
+    @SerialName("repeating_tasks") val repeatingTasks: Map<String, RepeatingTaskStatistics>,
 ) {
     companion object {
         @JvmStatic
         fun fromStats() = ScanStatistics(
             tasks = Stats.getTasks().mapValues { entry ->
                 entry.value.map { TaskStatistics.fromTaskStats(it) }
             },
-            repeatingTasks = Stats.getRepeatingTasks().mapValues { repeatingTasks ->
-                repeatingTasks.value.map { resultMap ->
-                    RepeatingTaskStatistics.fromRepeatingTaskStats(
-                        result = resultMap.key,
-                        repeatingTaskStats = resultMap.value,
-                    )
-                }
+            repeatingTasks = Stats.getRepeatingTasks().mapValues { repeatingTask ->
+                RepeatingTaskStatistics(
+                    executions = repeatingTask.value.map { resultMap -> resultMap.value.executions }
+                        .sum()
+                )
```

Comment:
we're dropping a lot of information here, like distinguishing between successful and failed executions, as well as the timing of each result. We may want to consider adding this back in the future.

Reply:
Should we track this somewhere? Maybe with a TODO?

Reply:
We can wait until we have a use case for it.